packages/server-core/src/media/storageprovider/storageprovider.interface.ts

```diff
@@ -202,12 +198,14 @@ export interface StorageProviderInterface {
    */
   doesExist(fileName: string, directoryPath: string): Promise<boolean>
 
+  getCacheDomain(internal?: boolean): string
+
   /**
-   * Get the object from edge cache, otherwise returns getObject.
+   * Get the object's URL on the edge cache.
    * @param key Key of object.
-   * @returns {StorageObjectInterface}
+   * @returns {string}
```

Comment:
this typedoc comment can be removed as return type is inferred by typescript

Reply:
the rest of the functions here are the same, we can do this another time